test/jasmine/assets/has_webgl_support.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var getContext = require('webgl-context')
+var getContext = require('webgl-context');
```

Comment:
There is [a rule](https://github.com/plotly/plotly.js/blob/master/.eslintrc#L25) which highlights literally every line, as far as I use windows :laughing: